### ISSUE TITLE
Is it possible to use the K150 to write the EEPROM/Flash of non-PIC chips?

### ISSUE THREAD

**Issue by danwdart** [Salamek/picpro#6]
Perhaps the ATmega644P, as it fits into the programmer and can itself be programmed, it just needs to receive the correct signals on the right pins.

**Comment by Salamek** [comment]
I don't think it is, as i understand it K150 programmer was designed to program PIC chips only, there is a firmware running directly on K150 that is doing all the "correct signals on the right pins" so maybe after changing this firmware? but there may be hardware limitations that may prevent it... not sure i'm not designer of K150 after all...

**Comment by danwdart** [comment]
Fair enough, thanks for quickly answering!
